Cython/Debugger/Tests/test_libcython_in_gdb.py

```diff
@@ -124,10 +124,10 @@ def test_CythonFunction(self):
                          'codefile.SomeClass.spam')
         self.assertEqual(self.spam_func.module, self.module)
 
-        assert self.eggs_func.pf_cname, (self.eggs_func, self.eggs_func.pf_cname)
-        assert not self.ham_func.pf_cname
-        assert not self.spam_func.pf_cname
-        assert not self.spam_meth.pf_cname
+        self.assertNotEqual(len(self.eggs_func.pf_cname), 0, (self.eggs_func, self.eggs_func.pf_cname))
+        self.assertEqual(len(self.ham_func.pf_cname), 0)
+        self.assertEqual(len(self.spam_func.pf_cname), 0)
+        self.assertEqual(len(self.spam_meth.pf_cname), 0)
```

Comment:
This changes the semantics. Previously, we didn't require the values to be strings (or something with a length). Now we do. Not sure if this was intended.

Reply:
I can of course just use assertTrue, but it seems to me better to clarify.  Do we want to use anything other than a string / byte for this field?